airbyte-integrations/connectors/source-postgres/integration_tests/abnormal_state.json

```diff
@@ -3,15 +3,17 @@
     "type": "STREAM",
     "stream": {
       "stream_state": {
-        "stream_name": "id_and_name",
-        "stream_namespace": "public",
+        "version": 2,
+        "state_type": "cursor_based",
+        "stream_name": "id_and_name_hook",
         "cursor_field": ["id"],
         "cursor": "4",
-        "cursor_record_count": 1
+        "cursor_record_count": 1,
+        "stream_namespace": "%s"
       },
       "stream_descriptor": {
-        "name": "id_and_name",
-        "namespace": "public"
+        "name": "id_and_name_hook",
```

Comment:
just to resolve the discover error we are facing on postgres acceptance test! it is looking for id_and_name table. If I did not rename this, all streams will have id_and_name table and discover would fail because it found multiple streams with the same name (which I think it needs to be fixed, but this is just to bypass and unblock the CI)

Reply:
That's interesting... let's file another ticket to track this separately.